.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: go
 go:
-  - 1.2
-  - 1.3
-install: go get code.google.com/p/go.tools/cmd/cover
+  - 1.7
+  - 1.8
+install: go get golang.org/x/tools/cmd/cover
```

Comment:
You should able to drop this line entirely, cover no longer requires to be installed.